src/query/query_executor.c

```diff
@@ -20922,14 +20922,6 @@ qexec_execute_build_indexes (THREAD_ENTRY * thread_p, XASL_NODE * xasl, XASL_STA
 	}
     }
 
-  for (i = 0; i < rep->n_attributes; i++)
-    {
-      if (attr_names[i] != NULL)
-	{
-	  free_and_init (attr_names[i]);
-	}
-    }
-
```

Comment:
It is not :) . There was a bad free  in `mr_readval_string_internal`, not here. I am reverting this and pushing afterwards the change.